SPECS/libtool/libtool.spec

```diff
@@ -79,16 +88,31 @@ make %{?_smp_mflags} check
 %{_libdir}/libltdl.so.7.3.1
 
 %changelog
+* Mon Aug 30 2021 Bala <balakumaran.kannan@microsoft.com> - 2.4.6-8
+- Merged to 1.0 from dev branch
```

Comment:
Not something to mention inside the spec - these should be agnostic to how our repo is organized.